src/modules/DeepLinkingManager.js

```diff
@@ -33,11 +35,31 @@ class DeepLinkingManager extends React.Component<Props> {
   componentDidUpdate () {
     if (Object.keys(this.props.wallets).length > 0 && this.props.deepLinkPending) this.checkForWallet()
   }
+  processPluginDeepLink = (parsedUrl: Object) => {
+    if (parsedUrl.pathname.includes('simplex')) {
```

Comment:
Where do we take the `pathname` and inject it into the plugin? How do they know what the URI parameters are?